package.json

```diff
@@ -107,15 +107,13 @@
     "@gooddata/js-utils": "3.4.0",
     "@gooddata/numberjs": "^3.1.2",
     "@gooddata/typings": "^2.13.2",
-    "@types/highcharts": "4.2.56",
     "ag-grid-community": "20.0.0",
     "ag-grid-react": "20.0.0",
     "classnames": "^2.2.6",
     "custom-event": "^1.0.1",
     "fixed-data-table-2": "0.7.17",
-    "highcharts": "4.2.5",
+    "highcharts": "7.1.1",
```

Comment:
**Note**: If you would be in time pressure, feel free to skip this comment, we'll do it globally in FET-159

Please consider adding highcharts dependency to ^7.1.1, the process is described in FET-159, copying adjusted steps for just one dependency (highcharts) below:

1. run `yarn add highcharts@^7.1.1` (notice the @^)
2. check the diff in yarn.lock and see if the version has not been updated (i.e. only the first line of the dependency block should change), revert any other changes in the block for `dependency` and also revert changes for other packages
3. commit this

Reply:
https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/2162/artifact/test-storybook/html-report/index.html

I tried but many storybook test fail. It could introduce more bugs. So we should not apply caret for Highcharts.